User Photo Feed: Completed HTML markdown

mserykh opened the pull request:
# Related Issue
[User Photo Feed: Create an HTML semantic layout](https://github.com/mserykh/instaclone/issues/8)

# Proposed Changes
- Created a directory with essential images and icons
- Created a markdown file for the `User Photo Feed` page

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+                <li><a href="#">Language</a></li>
+            </ul>
+        </nav>
+        <span>© 2020 Instagram from Facebook</span>
```
Please remove the copyright

mserykh replied:
@dev-experience OK

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+        </nav>
+
+        <article class="user-photo-feed">
+            <div><a href="#"><img src="./assets/user-photo-feed/img/user-picture-1.jpg" height="293" width="293" alt=""></a></div>
```
Isn't it a list?

mserykh replied:
To be honest I am not sure.

mserykh replied:
@dev-experience Is it better to put them into a list?

dev-experience replied:
> @dev-experience Is it better to put them into a list?

Semantically it's a list.

mserykh replied:
@dev-experience OK, I will make it a list

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+    <main>
+        <h1></h1>
+
+        <section>
```
I believe, `section` should be nested into an article or something.

mserykh replied:
@dev-experience  Not always. It could go separately.

dev-experience replied:
Doesn't look good for me. But if you insist...

mserykh replied:
@dev-experience I will leave it as is

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+
+        <section>
+            <header>
+                <img src="./assets/user-photo-feed/img/profile_picture.jpg" height="150" width="150" alt="">
```
Empty `alt`?

mserykh replied:
@dev-experience Yes, for now they are empty everywhere on the page. Should I add text?

dev-experience replied:
`alt` is meant to have a text, yes.

mserykh replied:
@dev-experience I will add texts

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+        </section>
+        <div class="highlights">
+            <ul class="highlights-list">
+                <li class="highlights-item">
```
BEM?

mserykh replied:
@dev-experience Not yet. I was thinking to apply it in CSS.

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+        </nav>
+
+        <article class="user-photo-feed">
+            <div><a href="#"><img src="./assets/user-photo-feed/img/user-picture-1.jpg" height="293" width="293" alt=""></a></div>
```
Why there are CSS classes in the highlights but not here?

mserykh replied:
@dev-experience I added some classes for now. I was thinking to work closer with styles while I will work on CSS.

dev-experience replied:
Doesn't look very consistent.

mserykh replied:
@dev-experience Agree. OK, I will add classes.

dev-experience commented on src/user-photo-feed.html:
```diff
@@ -0,0 +1,122 @@
+    </header>
+
+    <main>
+        <h1></h1>
```
??

mserykh replied:
@dev-experience I forgot about `h1`. There is no `h1` on the page so there must be one. I will create it and add to it "invisible" class.

mserykh commented:
I made requested changes:
* Added texts to all `alt` atributes
* Added classes to all elements
* Deleted copyright from the footer
* Added text and class to `h1`
* I wrapped photo feed in a list `ul`
I also adjusted layout with the following:
* Added elements to every photo in the feed
* Added icons to the menu
* Added a wrapper to a main section
